javascript/ql/test/query-tests/Security/CWE-798/HardcodedCredentials.js

```diff
@@ -155,3 +155,7 @@
         }
     });
 })();
+
+(function(){
+	require("cookie-session")({ secret: "change_me" });
```

Comment:
Hm, I can totally see why we don't want to flag these in "sample projects" but in production this would be vulnerable.

As I understand it, secrets like these are devious in that if you forget to change it, *everything will appear to work*, so they're a bit different from actual credentials in that way.

Reply:
Right. I have removed the "key" credential kind from the whitelist.